gallery/tutorials/pipeline_demo.py

```diff
@@ -125,60 +122,78 @@
 # with noise and CTF corruption
 src.images[0:10].show()
 
+# %%
+# Image Preprocessing
+# -------------------
+# We apply some image preprocessing techniques to prepare the
+# the images for denoising via Class Averaging.
+
+# %%
+# Downsampling
+# ------------
+# We downsample the images to remove high frequency noise and improve the
```

Comment:
I'm not sure we're removing high frequency noise exactly (its a little more complicated.. perhaps averaging, alas less words is probably more here).  Maybe just state `reducing the image size will improve...`

Reply:
Sounds good. Done.